Standards/CCSMB-13.md

```diff
@@ -0,0 +1,47 @@
+# CCSMB-13: Stereo DFPWM Audio Protocol
+
+*Author: @Drucifer (Discord)*
+
+*Version: 0.1*
+
+*Last revised: YYYY-MM-DD*
+
+## Quick information
+
+| Information |                             |
+| ----------- | --------------------------- |
+| Version     | 0.1                         |
+| Author      | @Drucifer (Discord)         |
+| Type        | Stereo DFPWM Audio Protocol |
+| MIME        | `audio/sdap`                |
+
+## Introduction
+
+This standard serves to provide framework for unifying and informing the layout of transmissions of stereo DFPWM content on modem channels to increase access to the general public while preserving the hobbyist usage.
+
+## Technical details
+
+Stereo DFPWM Audio Protocol (SDAP) has two functional rulesets depending on the modem used:
+| Modem Type | Transmission Mode |
+| ---------- | ----------------- |
+| Ender      | Full-Power        |
+| Wireless   | Low-Power         |
+| Wired      | Low-Power/Wired   |
+
+
+*Important: SDAP does not use Rednet! SDAP is a direct-modem protocol! Broadcasting SDAP packets via Rednet on a public server may result in bad things happening and I won't be held responsbile. Always make sure you're using direct modem calls, not Rednet!*
```

Comment:
I propose we define network encapsulation and such so we can either exclude this from it or make sure it wont conflict.

Reply:
I defined modem usage and split off this notice under the subheader Transmission medium(s)